pset-iterators.js

```diff
@@ -184,6 +221,16 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+const shortTermPresidents = presidents.filter(president => {
```

Comment:
@anniecdavis27 you're on the right path here, you want to capture the startDate and the endDate for each president and then have a condition that only matches against presidents that have an endDate - startDate < 4 

Check the solution for more details: 
https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L204